incubator/hnc/pkg/reconcilers/hierarchical_namespace.go

```diff
@@ -98,7 +104,7 @@ func (r *HierarchicalNamespaceReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 // the namespace instances to update its HNS state. This will be the only place the "Owner"
 // field in the forest is set. Therefore, the "Owner" field can be used in the forest to get
 // all the HNS objects of a namespace.
-func (r *HierarchicalNamespaceReconciler) syncWithForest(log logr.Logger, inst *api.HierarchicalNamespace, nsInst *corev1.Namespace, hcInst *api.HierarchyConfiguration) {
+func (r *HierarchicalNamespaceReconciler) syncWithForest(log logr.Logger, inst *api.HierarchicalNamespace, nsInst *corev1.Namespace, hcInst *api.HierarchyConfiguration) (createNS bool) {
```

Comment:
Maybe replace `inst` with `hnsInst` because there are three instances floating around here with nearly equal importance.

Reply:
Removed `hcInst` in HNSR, so kept using `inst` here in the HNSR.